src/main/web/templates/auth-needed.ftl

```diff
@@ -0,0 +1,3 @@
+<html>
+You are authenticated3!
```

Comment:
Typo

Reply:
I think I was testing out to see if changes were picked up at one point. Removed.